apps/govern/components/VeOlas/VeOlasManage.tsx

```diff
@@ -0,0 +1,68 @@
+import { Button, Col, Row } from 'antd';
+import { useState } from 'react';
+
+import { notifySuccess } from '@autonolas/frontend-library';
+
+import { notifyError } from 'libs/util-functions/src';
+
+import { withdrawVeolasRequest } from 'common-util/functions';
+import { useFetchBalances } from 'hooks/index';
+
+import { useVeolasComponents } from './useVeolasComponents';
+
+export const VeOlasManage = () => {
+  const { isLoading, canWithdrawVeolas, account, refetch } = useFetchBalances();
+  const {
+    getBalanceComponent,
+    getVotingPowerComponent,
+    getVotingPercentComponent,
+    getLockedAmountComponent,
+    getUnlockTimeComponent,
+    getUnlockedAmountComponent,
+  } = useVeolasComponents();
```

Comment:
need to read the hook, but not sure why we would return these from a hook?
why not just import the components directly?
(will read hook and figure it out shortly)